apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/cert/ApolloCertificate.java

```diff
@@ -28,14 +28,17 @@
     private final X509Certificate certificate;
     private final CertAttributes va  = new CertAttributes();
 
-    public static ApolloCertificate loadPEMFromPath(String path) throws FileNotFoundException, ApolloCertificateException {
+    public static ApolloCertificate loadPEMFromPath(String path) throws  ApolloCertificateException, IOException {
+        return ApolloCertificate.loadPEMFromStream(new FileInputStream(path));
+    }
+    
+    public static ApolloCertificate loadPEMFromStream(InputStream is) throws IOException, ApolloCertificateException {
```

Comment:
incoming InputStream should be closed in outside code.

Reply:
It will be closed on scope exit anyway